core/src/main/java/org/hibernate/ogm/cfg/OgmConfiguration.java

```diff
@@ -37,81 +23,14 @@
  */
 public class OgmConfiguration extends Configuration implements Configurable {
```

Comment:
`OgmConfiguration` is still there, but it only overrides `buildSessionFactory` to expose the proper OGM type. All the actual customizations are done via the integration points in the new bootstrap SPI.